packages/expo-dev-menu/package.json

```diff
@@ -60,8 +60,6 @@
     "expo-dev-client-components": "1.8.1",
     "expo-module-scripts": "^3.0.0",
     "fuse.js": "^6.4.6",
-    "graphql": "^15.3.0",
-    "graphql-tag": "^2.10.1",
```

Comment:
Just pointing this out since this is more of a drive by change.
I think most of these deps are unused and have moved to `expo-dev-launcher` potentially?
hence I'm removing at least the relevant `graphql` and `graphql-tag` installations here. This didn't affect users anyway though since it's in `devDependencies`